source/agora/consensus/PreImage.d

```diff
@@ -572,6 +577,62 @@ public void getCycleSeed (in KeyPair pair, in uint validator_cycle, out Hash see
     }
 }
 
+version (unittest)
+private void getCycleSeed1008 (in KeyPair pair, in uint validator_cycle, out Hash seed,
```

Comment:
Perhaps it would be cleaner to use an enum for the supported validator_cycles in test code and pass that as a param to getCycleSeed.

Reply:
Right, that would be more robust. I'll make an issue about it with what you suggested in [this comment](https://github.com/bosagora/agora/pull/2291#pullrequestreview-713406152).